### DISCUSSION TITLE
Applicationset for single helm chart with many values files

### DISCUSSION THREAD

**Discussion by mppraxis** [argoproj/argo-cd#16997] [Q&A] [answered]
I would like to create an ApplicationSet that points to a single helm chart with many value files, and have it create an application per value file. Is this possible?

I have a working app-of-apps doing this, but would like to simplify if possible.

**Comment by gdsoumya** [comment, marked as answer]
You could try using [appset list generator](https://argo-cd.readthedocs.io/en/stable/operator-manual/applicationset/Generators-List/) where you provide the list of helm files you want to use and then in the app template keep the same repo details but use the helm file from the list.

**Reply by mppraxis** [reply]
That works, thank you.
